src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -255,8 +255,8 @@ export const TreeItem: React.FC<TreeItemProps> = props => {
       ) : null}
       {isTitleHovered && !processingEntity.processingType ? (
         <>
-          {canPreview(relativePath) && !isInPreviewMode && (
```

Comment:
Should we see the Preview label when in preview mode? The file explorer is disabled and it's confusing if you can still interact with the `Preview` action. Maybe make it disabled or just don't show it?

Reply:
In Preview mode we also can access sub menu of item, should I hide it too then?

Reply:
i think the general approach is to disable - not to hide - @tommywhitehead ?

Reply:
I've disabled this button in Preview mode, "..." still allow to click on it in preview mode. 
Intention of this feature is to mark file as something that we can preview in Navigator